examples/example.rs

```diff
@@ -52,7 +52,8 @@ fn main() -> rustyline::Result<()> {
     env_logger::init();
     let config = Config::builder()
         .history_ignore_space(true)
-        .completion_type(CompletionType::List)
+        .completion_type(CompletionType::Fuzzy)
```

Comment:
Please, don't use Fuzzy.

Reply:
Oh whoops sorry, this was an experiment that wasn't supposed to go in the PR.